src/app/util/af-types.h

```diff
@@ -415,6 +415,11 @@ struct EmberAfDefinedEndpoint
      * endpoint
      */
     chip::DataVersion * dataVersions = nullptr;
+
+    /**
+     * Root endpoint id for composed device type.
+     */
+    chip::EndpointId composedEpId = chip::kInvalidEndpointId;
```

Comment:
I think the term `composedxx` isn't quite conveying the fact that this is not denoting the set of child endpoints from a given parent, but the reverse: this is the endpoint ID of the parent.

`parentEndpointId` or something?

Reply:
`parentEndpointId ` is well.